src/taipy/gui/icon.py

```diff
@@ -45,14 +46,29 @@ def __init__(self, path: str, text: t.Optional[str] = None) -> None:
                 the application directory.
             text (Optional[str]): The text associated to the image. If _text_ is None,
                 there is no text associated to this image.
+            light_path (Optional[str]): The path to the light theme image (fallback to _path_ if not defined).
```

Comment:
Please use '*' for parameter name references instead of '_'.

Reply:
will do